xchange-binance/src/main/java/org/knowm/xchange/binance/dto/trade/BinanceNewOrder.java

```diff
@@ -2,8 +2,11 @@
 
 import com.fasterxml.jackson.annotation.JsonProperty;
 import java.math.BigDecimal;
+import java.util.Date;
+import org.knowm.xchange.binance.BinanceAdapters;
+import org.knowm.xchange.dto.trade.LimitOrder;
 
-public final class BinanceNewOrder {
+public final class BinanceNewOrder extends LimitOrder {
```

Comment:
binances order does not have the same field names as our LimitOrder, remove this extends
DTO are NOT mappers